tests/testthat/test_model_run.R

```diff
@@ -36,12 +36,11 @@ test_that("can run model and return results", {
   )
 
   expect_named(res, c(
-    "runId",
     "parameters",
     "costs",
-    "capacities",
+    "time_series",
```

Comment:
This test has changed because we're no longer reading in the dummy data file, from which the result list originally took its names. The runId is added when the results are fetched (model_run doesn't have access to it).